Drop-In/src/main/java/com/braintreepayments/api/dropin/AddCardActivity.java

```diff
@@ -378,23 +378,9 @@ public void onError(Exception error) {
         }
     }
 
-    @Override
-    public boolean onCreateOptionsMenu(Menu menu) {
```

Comment:
Is deleting this public method technically a breaking change?

Reply:
In this case it should be fine, `onCreateOptionsMenu` is a method on the `AppCompatActivity` super class.

Reply:
Yeh - I wasn't sure if since we had some custom logic inside our override that removing our override was technically a breaking change.

Reply:
Yeah I know removing CardIO was a breaking change for `android-card-form`. I wonder if it makes sense to major version bump drop-in since CardIO is no longer there 🤔

Reply:
Hm - I think since it isn't making incompatible API changes for braintree-android-drop-in users, it doesn't qualify for a major bump.

Reply:
I dunno - it's removing pretty significant functionality. I think it would qualify for a major version bump.

Reply:
It's definitely a breaking change to drop Card.io, so whether this particular change is breaking or not shouldn't matter. We will need to do a major version bump regardless.

Reply:
Agreed.

Reply:
I'm not too familiar with the changes here, but does anything change with the merchant with this removal? If so, it should be listed in the changelog under the breaking changes section.

Reply:
Yeah, there is a superclass implementation that provides a default home button in the Android [Action Bar](https://developer.android.com/training/appbar). It's [Activity#createOptionsMenu](https://developer.android.com/reference/android/app/Activity.html#onCreateOptionsMenu(android.view.Menu)).

We were adding a _Scan Card_ icon as an additional menu item to activate card.io. We could mention that this button was removed, but merchants shouldn't have to take additional action here.